### PR TITLE
chore: add policy to publish new npm package

### PR DESCRIPTION
**Pull request by mmarchini**

### CODE REVIEW

**Comment by mmarchini** [comment]
cc @nodejs/tsc @nodejs/community-committee

**Comment by MintThitiratChamnan** [comment]
> 

Ok

File: npm-management.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ well.
 The credentials required for the `nodejs-foundation` user are maintained in
 encrypted form in the [secrets repo][].
 
+## Publishing new packages
+
+To publish a new npm package, send a request to nodejs/admin, pinging TSC and
```
Comment (bnb) [inline npm-management.md:42]:
```suggestion
To publish a new "official" npm package for the first time from a repository under the `nodejs/` GitHub organization, send a request to nodejs/admin, pinging TSC and
```

honestly just suggesting this to get it wrapped up - we probably don't need to continue having this open, it's pretty good and can be further modified as the need arises.

Comment (Trott) [reply npm-management.md:42]: No scare-quotes around _official_:

```
To publish a new official npm package for the first time from a repository under the `nodejs/` GitHub organization, send a request to nodejs/admin, pinging TSC and
```

Comment (Trott) [reply npm-management.md:42]: And I agree this can land as-is and be iterated on.

File: npm-management.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ well.
...
+CommComm. Publishing a new package follows the same rules as
```
Comment (Trott) [inline npm-management.md:43]:
```suggestion
Publishing a new package follows the same rules as 
```

Comment (ljharb) [inline npm-management.md:43]:
```suggestion
To publish a new npm package, send a request to nodejs/admin, pinging TSC. Publishing a new package follows the same rules as 
```

File: npm-management.md (hunk 1 above)
Comment (Trott) [reply npm-management.md:42]:
```suggestion
To publish a new npm package, send a request to nodejs/admin, pinging TSC.
```

File: npm-management.md [hunk 3]
```diff
@@ -37,6 +37,12 @@ well.
 The credentials required for the `nodejs-foundation` user are maintained in
 encrypted form in the [secrets repo][].
 
+## Publishing new packages
+
+To publish a new npm package, send a request to nodejs/admin, pinging TSC.
```
Comment (Trott) [inline npm-management.md:42]:
```suggestion
To publish a new npm package, open an issue in nodejs/admin, pinging TSC.
```

**Review by Trott — APPROVED** [review]

**Review by jasnell — APPROVED** [review]

**Review by bnb — APPROVED** [review]

**Review by WaleedAshraf — APPROVED** [review]

**Pull request merged**